package.json

```diff
@@ -21,14 +21,13 @@
     "chai": "^4.2.0",
     "coveralls": "^3.0.6",
     "jshint": "^2.10.2",
-    "mocha": "^6.2.0",
-    "nyc": "^14.1.1",
-    "q-io": "^1.13.6"
+    "mocha": "^8.3.1",
+    "nyc": "^14.1.1"
```

Comment:
Updated nyc to pick up a newer yargs-parser module for (this advisory)[https://npmjs.com/advisories/1500].